open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/graph/LineageGraphConnector.java

```diff
@@ -858,5 +813,23 @@ public LineageVertexResponse getEntityDetails(String guid) {
     public boolean isEntityInGraph(String guid) {
         return !g.V().has(PROPERTY_KEY_ENTITY_GUID, guid).toList().isEmpty();
     }
+
+    /**
+     * commit the graph transaction if graph transactions are enabled
+     * @param g the graph traversal
+     */
+    private void commitTransaction(GraphTraversalSource g) {
+        if (graphFactory.isSupportingTransactions()) {
```

Comment:
I think this refactoring change is good, as you are refactoring - I wonder if you could put these methods in a place where the other Egeria code issuing commits and rollback like this could call them. I notice that the other callers of commit and rollback  in Egeria do not have the graphFactory.isSupportingTransactions() check. We could then start to accumulate common code that users of the graph call.

Reply:
Hey David, I think it would be a good idea to tackle your proposal in a separate issue/PR. What do you think?